README.md

```
@@ -83,43 +98,57 @@ order := sveawebpay.RecurOrder{
 ## Get information about a transaction
+
 Get information about a specific order with either the specified transaction ID or customer reference number. Both does not need to be provided. If `transactionID` is less than or equal to `0`, `customerRefNo` will be used. If `customerRefNo` is an empty string as well a package error will be returned. If both are set `transactionID` will be used.
 
 **Please only use this when needed. Repetitive polling is not allowed.**
+
 ```go
-transaction, err := c.GetTransaction(transactionID, customerRefNo)
-```
+response, err := c.GetTransaction(transactionID, customerRefNo)
+transaction := response.Transaction
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)